exercises/fulfill_a_promise/problem.md

```diff
@@ -16,43 +16,65 @@ the syntax is.  We are about to dive into that in detail.
 
 ## Setup
 
-To do many of the lessons in this workshop, you will need to install
-the `q` module.  To do this, type the following in your project's
-directory:
+To do many of the lessons in this workshop, you will need an installation of
+Node.js that supports ECMAScript 6 promises. That includes Node.js 0.12 or
+higher, and all versions of io.js.
+
+To find out if your Node.js installation has native promise support, execute
+the following in a Node.js REPL:
+
+```js
+typeof Promise !== "undefined"
+```
+
+If the REPL returns "true", then you are good to go! If not, you’ll have to use
+one of the many shims people have implemented. For this tutorial, we recommend
+`es6-promise` which aims to be strictly compliant to ES6 without any extra
+features. To use `es6-promise`, execute the following in a shell, preferably in the directory you’ll write your programs in:
```

Comment:
Mind your wrapping. It's rather jarring when this line shows up in the terminal.